apps/client/src/components/Conclusion.js

```diff
@@ -78,42 +69,24 @@ const Conclusion = ({ checker, matomoTrackEvent }) => {
   return (
     <>
       <Paragraph>
-        Op basis van uw antwoorden vindt u hieronder wat voor uw activiteit van
-        toepassing is.
+        U bent klaar met de vergunningcheck. Dit is de uitkomst:
       </Paragraph>
 
-      {displayConclusions.map(({ title, description }) => (
-        <Fragment key={title}>
-          <Heading forwardedAs="h2">{title}</Heading>
-          <Markdown eventLocation={sections.CONCLUSION} source={description} />
-        </Fragment>
-      ))}
-
-      <HideForPrint>
-        {needsPermit && !contactConclusion && (
-          <ComponentWrapper marginBottom={40}>
-            <Button
-              type="button"
-              color="secondary"
-              onClick={handlePermitButton}
-            >
-              Vergunning aanvragen
-            </Button>
-          </ComponentWrapper>
-        )}
-
-        {!isIE && !isMobile && (
-          <ComponentWrapper>
-            <PrintButton
-              color="primary"
-              onClick={handlePrintButton}
-              type="button"
-            >
-              Conclusie opslaan
-            </PrintButton>
-          </ComponentWrapper>
-        )}
-      </HideForPrint>
+      {contactConclusion && (
+        <ContactConclusion
+          contactConclusion={contactConclusion}
+          handlePrintButton={handlePrintButton}
```

Comment:
Would be nice if we could solve `handlePrintButton` in a different way so we don't have to drill props here.